src/EnergyPlus/Humidifiers.cc

```diff
@@ -742,31 +742,27 @@ namespace Humidifiers {
         // na
 
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        std::string ModuleObjectType; // for ease in getting objects
-        int RefrigerantIndex;         // refrigerant index
-        int WaterIndex;               // fluid type index
-        Real64 NominalPower;          // Nominal power input to humidifier, W
-        Real64 WaterSpecHeatAvg;      // specific heat of water, J/kgK
-        Real64 SteamSatEnthalpy;      // enthalpy of saturated steam at 100C, J/kg
-        Real64 WaterSatEnthalpy;      // enthalpy of saturated water at 100C, J/kg
-        bool IsAutoSize;              // Indicator to autosize
-        bool HardSizeNoDesRun;        // Indicator to a hard-sized field with no design sizing data
-        bool ErrorsFound(false);      // TRUE if errors detected in input
-        Real64 NomPowerDes;           // Autosized nominal power for reporting
-        Real64 NomPowerUser;          // Hardsized nominal power for reporting
-        Real64 MassFlowDes;           // Design air mass flow rate
-        Real64 InletHumRatDes;        // Design inlet humidity ratio
-        Real64 OutletHumRatDes;       // Design outlet humidity ratio
-        Real64 NomCapVolDes;          // Autosized Nominal capacity volume for reporting
-        Real64 NomCapVolUser;         // HardSized nominal capacity volume for reporting
-        Real64 AirVolFlow;            // Design air volume flow rate
-        Real64 AirDensity;            // Density of air
+        Real64 NominalPower;     // Nominal power input to humidifier, W
+        Real64 WaterSpecHeatAvg; // specific heat of water, J/kgK
+        Real64 SteamSatEnthalpy; // enthalpy of saturated steam at 100C, J/kg
+        Real64 WaterSatEnthalpy; // enthalpy of saturated water at 100C, J/kg
+        bool ErrorsFound(false); // TRUE if errors detected in input
+        Real64 NomPowerDes;      // Autosized nominal power for reporting
+        Real64 NomPowerUser;     // Hardsized nominal power for reporting
+        Real64 MassFlowDes;      // Design air mass flow rate
+        Real64 InletHumRatDes;   // Design inlet humidity ratio
+        Real64 OutletHumRatDes;  // Design outlet humidity ratio
+        Real64 NomCapVolDes;     // Autosized Nominal capacity volume for reporting
+        Real64 NomCapVolUser;    // HardSized nominal capacity volume for reporting
+        Real64 AirVolFlow;       // Design air volume flow rate
+        Real64 AirDensity;       // Density of air
 
         if (HumType == HumidType::Electric || HumType == HumidType::Gas) {
-            IsAutoSize = false;
-            HardSizeNoDesRun = false;
+            bool IsAutoSize = false;
+            bool HardSizeNoDesRun = false;
             NomPowerDes = 0.0;
             NomPowerUser = 0.0;
+            std::string ModuleObjectType; // for ease in getting objects
```

Comment:
[src/EnergyPlus/Humidifiers.cc:745]:(style),[variableScope],The scope of the variable 'ModuleObjectType' can be reduced.
[src/EnergyPlus/Humidifiers.cc:746]:(style),[variableScope],The scope of the variable 'RefrigerantIndex' can be reduced.
[src/EnergyPlus/Humidifiers.cc:747]:(style),[variableScope],The scope of the variable 'WaterIndex' can be reduced.
[src/EnergyPlus/Humidifiers.cc:752]:(style),[variableScope],The scope of the variable 'IsAutoSize' can be reduced.
[src/EnergyPlus/Humidifiers.cc:753]:(style),[variableScope],The scope of the variable 'HardSizeNoDesRun' can be reduced.